src/Bijectors.jl

```diff
@@ -141,71 +143,95 @@ end
 
 const SimplexDistribution = Union{Dirichlet}
 
-function link(d::SimplexDistribution, x::AbstractVector{T}) where T<:Real
+function link(d::SimplexDistribution, x::AbstractVector{T}, ::Type{Val{proj}} = Val{true}) where {T<:Real, proj}
```

Comment:
There are a few locations where the function headers exceed 92 characters. Please resolve this by splitting them up in accordance with the Turing style guide

Reply:
Sure. I will fix this and merge.